app/controllers/words_controller.rb

```diff
@@ -14,6 +14,28 @@ def show
     @reviews = @word.reviews
   end
 
+  def new
+    if current_user
+      @word = Word.new
+    else
+      flash[:errors] = "You must be logged in to create a word."
+      redirect_to root_path
+    end
+  end
+
+  def create
+    if current_user
```

Comment:
If you have a top-level `if` statement with no `else` in a controller action, what you actually want is a Controller Filter. Do some googling and see if you can suss it out!